core/domain/suggestion_jobs_one_off.py

```diff
@@ -348,3 +348,45 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class ContentSuggestionFormatUpdateJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Do we need a unit test for this new job? (Not sure why the backend test didn't fail for the coverage issue!)

Reply:
I missed [your comment](https://github.com/oppia/oppia/pull/12068#issuecomment-788750297), please ignore the above comment.